site/_js/firebase.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @fileoverview Firebase integration.
+ */
+import {initializeApp} from 'firebase/app';
+//import { initializeAppCheck, ReCaptchaV3Provider } from "firebase/app-check";
+import {getDatabase, ref, runTransaction, onValue} from 'firebase/database';
```

Comment:
`firebase/database` refers to the [Firebase Realtime Database](https://firebase.google.com/docs/database/web/start), but our suggestion was to use the [Cloud Firestore](https://firebase.google.com/docs/firestore) instead.

Could you explain why you went with the Firebase Realtime Data instead? If there's a reason, based on the [criteria outlined in the docs](https://firebase.google.com/docs/database/rtdb-vs-firestore), then we can reevaluate, but I want to make sure it was not just chosen in error.

Reply:
Because of attached reasons we have chosen Real time Database instead of Firestore - 
![image](https://user-images.githubusercontent.com/101087073/160235291-2704c70b-2c5e-4b9f-803e-08d09a59a0dc.png)

Please share your thoughts!